src/shared/components/lollipopMutationPlot/LollipopMutationPlot.tsx

```diff
@@ -414,13 +414,12 @@ export default class LollipopMutationPlot extends React.Component<ILollipopMutat
                             <button className="btn btn-default btn-xs" onClick={this.handlers.handleSVGClick}>
                                 SVG <i className="fa fa-cloud-download" aria-hidden="true"></i>
                             </button>
-
-
+                            /* TODO: output real PDF instead of embedded picture PDF 
```

Comment:
This PDF on mutation tab is real. I think we want to disable the one on the Cancer Summary tab for now.

Reply:
Thanks @onursumer wrong one